backend/src/main/java/ca/bc/gov/app/exception/InvalidAccessTokenException.java

```diff
@@ -6,7 +6,11 @@
 
 @ResponseStatus(HttpStatus.UNAUTHORIZED)
 public class InvalidAccessTokenException extends ResponseStatusException {
+
+  private static final long serialVersionUID = -1534366921974396495L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data